Index: .env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>MONGO_LOGIN=admin\r\nMONGO_PASSWORD=admin\r\nMONGO_HOST=127.0.0.1\r\nMONGO_PORT=27017\r\nMONGO_AUTHDATABASE=admin
===================================================================
diff --git a/.env b/.env
--- a/.env	
+++ b/.env	
@@ -1,5 +1,6 @@
 MONGO_LOGIN=admin
 MONGO_PASSWORD=admin
+#MONGO_HOST=45.12.73.253
 MONGO_HOST=127.0.0.1
 MONGO_PORT=27017
-MONGO_AUTHDATABASE=admin
\ No newline at end of file
+MONGO_AUTHDATABASE=admin
